opencraft/urls.py

```diff
@@ -51,3 +52,4 @@
     urlpatterns = [
         url(r'^__debug__/', include(debug_toolbar.urls)),
     ] + urlpatterns
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Could you explain in the commit msg how exactly this improves the devstack setup?

Reply:
This is just to make testing easier. The devstack doesn't set up the Django media assets path (so all urls from uploaded files are broken). To avoid having to deal with this headache while testing I've added this line.